docs/explanations/motd_messages.md

```diff
@@ -28,7 +54,10 @@ Expanded Security Maintenance for Applications is enabled.
 To see these additional updates run: apt list --upgradable
 ```
 
-However, if we were running this on an ESM series, we would instead see
+### Machine is fully attached, but already in ESM
```

Comment:
I got a bit confused by the wording in this section. I wasn't sure why, if the machine is attached and in ESM, why only ESM infra being advertised. I asked about this earlier and it was confirmed that whichever services are enabled get shown but I'm not sure that's clear here. We probably want to revise this wording to make sure it reflects what's currently happening after all the changes to terminology.

(I should clarify, I attached the comment to the sub-heading and not the paragraph below it by mistake, sorry!)